x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -176,6 +176,7 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
       core,
       endpointAppContext: endpointContext,
       kibanaIndex: globalConfig.kibana.index,
+      signalsIndex: config.signalsIndex,
```

Comment:
If I am reading this correctly, you are going to grab the signals prefix for _all_ spaces such as this default prefix:

```ts
.siem-signals
```

And then each space is going to create an index like so:

Default space:
```ts
.siem-signals-default
```

If I made a space called `test` it would end up like this:

```ts
.siem-signals-test
```

So, later on as long as you do this in telemetry:

```ts
${signalsIndex}*
```

or

```ts
${signalsIndex}-*
```

You will get all the signals from all the spaces. I don't think I saw that below though looking through the code so just wanted to verify if this works or if I'm missing something here.

Reply:
Thanks! Updated in 0c91a78